frontend/README.md

```diff
@@ -18,7 +17,7 @@ yarn install
 
 #### Contracts
 
-To start a local blockchain using `flow emulator`:
+To start a local blockchain & dev-wallet:
```

Comment:
it looks like the `yarn chain` command doesnt work  anymore and that script is now in `/backend` so these instructions should be in the backend repo?